openpype/hosts/maya/plugins/create/create_review.py

```diff
@@ -29,6 +38,43 @@ class CreateReview(plugin.MayaCreator):
     useMayaTimeline = True
     panZoom = False
 
+    # Overriding "create" method to prefill values from settings.
+    def create(self, subset_name, instance_data, pre_create_data):
+
+        members = list()
+        if pre_create_data.get("use_selection"):
+            members = cmds.ls(selection=True)
+
+        project_name = get_current_project_name()
+        asset_doc = get_asset_by_name(project_name, instance_data["asset"])
+        task_name = get_current_task_name()
```

Comment:
New style creators should be getting this information from the Creator's context. Right @iLLiCiTiT ?